### PR TITLE
[azure-storage] Added release notes for blob, file and queue

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/HISTORY.md [hunk 1]
```diff
@@ -5,6 +5,48 @@
 For release notes and more information please visit
 https://aka.ms/azure-sdk-preview1-python
 
+- **Breaking** New API desgin:
+    - Operations are now scoped to a particular client:
+        - `BlobServiceClient`: This client handles account-level operations. This includes managing service properties and listing the containers within an account.
+        - `ContainerClient`: The client handles operations for a particular blob. This includes creating or deleting that container, as well as listing the blobs within that container and managing properties and metadata.
```
Comment (kristapratico) [inline sdk/storage/azure-storage-blob/HISTORY.md]: I think you meant: "..operations for a particular **container**."

File: sdk/storage/azure-storage-blob/HISTORY.md [hunk 2]
```diff
@@ -5,6 +5,48 @@
...
+        This BlobClient handles all blob types (block, page and append). Where operations can behave differently according to type (i.e. `upload_blob`) the default behaviour will be block blobs unless otherwise specified.
+        - `LeaseClient`: Handles all lease operations for both containers and blobs.
+
+        These clients can be accessed by navigating down the client hierarchy, or instantiated directly using URLs to the resource (account, container or blob).
+      For full details on the new API, please see reference documentation.
+    - Copy blob operations now return a polling object that can be used to check the status of the operation, as well as abort the operation.
+    - New module level operations for simple upload and download using a blob URL.
+    - Download operations now return a streaming object that can download data in multiple ways:
+        - Iteration: The streamer is an iterable object that will download and yield the content in chunks. Only supports single threaded download.
+        - `content_as_bytes`: Return the entire blob content as bytes. Blocking operation that suppots multi-threaded download.
```
Comment (kristapratico) [inline sdk/storage/azure-storage-blob/HISTORY.md]: Typo: suppots -> supports

File: sdk/storage/azure-storage-blob/HISTORY.md [hunk 3]
```diff
@@ -5,6 +5,48 @@
...
+        - All service errors will now use the base type: `azure.core.exceptions.HttpResponseError`
+        - The are a couple of specific exception types derived from this base type for common error scenarios:
+            - `ResourceNotFoundError`: The resource (e.g. queue, message) could not be found. Commonly a 404 status code.
+            - `ResourceExistsError`: A resource conflict - commonly caused when attempting to create a resource that already exists.
+            - `ResourceModifiedError`: The resource has been modified (e.g. overwritten) and therefore the current operation is in conflict. Alternatively this may be raised if a condition on the operation is not met.
+            - `ClientAuthenticationError`: Authentication failed.
+    - Operation `set_blob_properties` has been renamed to `set_http_headers`.
+    - Operations `get_blob_to_<output>` have been replaced with `download_blob`. See above for download output options.
+    - Operations `create_blob_from_<input>` have been replace with `upload_blob`.
+    - Opteration `create_blob` has been renamed to separate `create_page_blob` and `create_append_blob`.
```
Comment (kristapratico) [inline sdk/storage/azure-storage-blob/HISTORY.md]: Typo: Opteration -> Operation

File: sdk/storage/azure-storage-blob/HISTORY.md (hunk 2 above)
Comment (rakshith91) [inline sdk/storage/azure-storage-blob/HISTORY.md]: nitpick 101: typo at supports.

File: sdk/storage/azure-storage-file/HISTORY.md [hunk 4]
```diff
@@ -5,6 +5,42 @@
...
+        - `FileServiceClient`: This client handles account-level operations. This includes managing service properties and listing the shares within an account.
+        - `ShareClient`: The client handles operations for a particular share. This includes creating or deleting that share, as well as listing the directories within that share, and managing properties and metadata.
+        - `DirectoryClient`: The client handles operations for a particular directory. This includes creating or deleting that directory, as well as listing the files and subdirectories, and managing properties and metadata.
+        - `FileClient`: The client handles operations for a particular file. This includes creating or deleting that file, as well as upload and download data and managing properties.
+
+        These clients can be accessed by navigating down the client hierarchy, or instantiated directly using URLs to the resource (account, share, directory or file).
+      For full details on the new API, please see reference documentation.
+    - The copy file operation now return a polling object that can be used to check the status of the operation, as well as abort the operation.
```
Comment (rakshith91) [inline sdk/storage/azure-storage-file/HISTORY.md]: returns^ a polling?

File: sdk/storage/azure-storage-file/HISTORY.md [hunk 5]
```diff
@@ -5,6 +5,42 @@
...
+        - The are a couple of specific exception types derived from this base type for common error scenarios:
+            - `ResourceNotFoundError`: The resource (e.g. queue, message) could not be found. Commonly a 404 status code.
+            - `ResourceExistsError`: A resource conflict - commonly caused when attempting to create a resource that already exists.
+            - `ResourceModifiedError`: The resource has been modified (e.g. overwritten) and therefore the current operation is in conflict. Alternatively this may be raised if a condition on the operation is not met.
+            - `ClientAuthenticationError`: Authentication failed.
+    - Operation `set_file_properties` has been renamed to `set_http_headers`.
+    - Operations `get_file_to_<output>` have been replaced with `download_file`. See above for download output options.
+    - Operations `create_file_from_<input>` have been replace with `upload_file`.
+    - Operations `get_share_acl` and `set_share_acl` have been renamed to `get_share_access_policy` and `set_share_access_policy`.
+    - Operation `set_share_properties` has been renamed to `set_share_quota`.
```
Comment (rakshith91) [inline sdk/storage/azure-storage-file/HISTORY.md]: Should it be `set_share_metadata`  and `set_share_quota` ?

File: sdk/storage/azure-storage-file/HISTORY.md [hunk 6]
```diff
@@ -5,6 +5,42 @@
...
+        - `FileClient`: The client handles operations for a particular file. This includes creating or deleting that file, as well as upload and download data and managing properties.
+
+        These clients can be accessed by navigating down the client hierarchy, or instantiated directly using URLs to the resource (account, share, directory or file).
+      For full details on the new API, please see reference documentation.
+    - The copy file operation now return a polling object that can be used to check the status of the operation, as well as abort the operation.
+    - The `close_handles` operation now return a polling object that can be used to check the status of the operation.
+    - Download operations now return a streaming object that can download data in multiple ways:
+        - Iteration: The streamer is an iterable object that will download and yield the content in chunks. Only supports single threaded download.
+        - `content_as_bytes`: Return the entire file content as bytes. Blocking operation that suppots multi-threaded download.
```
Comment (kristapratico) [inline sdk/storage/azure-storage-file/HISTORY.md]: Typo: Suppots -> Supports

File: sdk/storage/azure-storage-file/HISTORY.md (hunk 5 above)
Comment (annatisch) [reply sdk/storage/azure-storage-file/HISTORY.md]: No - set_share_metadata is still there, there used to be a set_share_properties operation that is now just set_share_quota

File: sdk/storage/azure-storage-queue/HISTORY.md [hunk 7]
```diff
@@ -5,6 +5,30 @@
 For release notes and more information please visit
 https://aka.ms/azure-sdk-preview1-python
 
+- **Breaking** New API desgin:
+    - Operations are now scoped to a particular client:
+        - `QueueServiceClient`: This client handles account-level operations. This includes managing service properties and listing the queues within an account.
+        - `QueueClient`: The client handles operations within a particular queue. This includes creating or deleting that queue, as well as enqueuing and dequeuing messages.
```
Comment (kristapratico) [inline sdk/storage/azure-storage-queue/HISTORY.md]: Should we use receiving instead of dequeuing messages to be consistent with the method?

Comment (annatisch) [reply sdk/storage/azure-storage-queue/HISTORY.md]: It's true the method is called receiving, but "dequeuing" I think consists of both "receiving and deleting" which is what you can do on the QueueClient... so a more "conceptual" description....

File: sdk/storage/azure-storage-blob/HISTORY.md [hunk 8]
```diff
@@ -2,8 +2,55 @@
...
+- Download operations now return a streaming object that can download data in multiple ways:
+    - Iteration: The streamer is an iterable object that will download and yield the content in chunks. Only supports single threaded download.
+    - `content_as_bytes`: Return the entire blob content as bytes. Blocking operation that supports multi-threaded download.
+    - `content_as_text`: Return the entire blob content as decoded text. Blocking operation that supports multi-threaded download.
+    - `download_to_stream`: Download the entire content to an open stream handle (e.g. an open file). Supports multi-threaded download.
+- New underlying REST pipeline implementation, based on the new `azure-core` library.
+- Client and pipeline configuration is now available via keyword arguments at both the client level, and per-operation. See reference documentation for a full list of optional configuration arguments.
+- Authentication using `azure-identity` credentials
+  - see this package's
+  [documentation](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/README.md)
```
Comment (chlowell) [inline sdk/storage/azure-storage-blob/HISTORY.md]: Probably ought to link to storage-blob's readme here.

File: sdk/storage/azure-storage-queue/HISTORY.md [hunk 9]
```diff
@@ -2,8 +2,37 @@
...
+    - `QueueClient`: The client handles operations within a particular queue. This includes creating or deleting that queue, as well as enqueueing and dequeueing messages.
+
+    These clients can be accessed by navigating down the client hierarchy, or instantiated directly using URLs to the resource (account or queue).
+    For full details on the new API, please see reference documentation.
+- New message iterator, for receiving messages from a queue in a continuous stream.
+- New underlying REST pipeline implementation, based on the new `azure-core` library.
+- Client and pipeline configuration is now available via keyword arguments at both the client level, and per-operation. See reference documentation for a full list of optional configuration arguments.
+- Authentication using `azure-identity` credentials
+  - see this package's
+  [documentation](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-keys/README.md)
```
Comment (chlowell) [inline sdk/storage/azure-storage-queue/HISTORY.md]: Similarly, storage-queue's readme would be better here.

File: sdk/storage/azure-storage-file/HISTORY.md [hunk 10]
```diff
@@ -2,8 +2,44 @@
 
 ## Version 12.0.0b1:
 
-For release notes and more information please visit
-https://aka.ms/azure-sdk-preview1-python
+Version 12.0.0b1 is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure Storage Files. For more information about this, and preview releases of other Azure SDK libraries, please visit
+https://aka.ms/azure-sdk-preview1-python.
+
+**Breaking changes: New API desgin**
```
Comment (chlowell) [inline sdk/storage/azure-storage-file/HISTORY.md]: des**gi**n -> design

File: sdk/storage/azure-storage-file/HISTORY.md [hunk 11]
```diff
@@ -2,8 +2,44 @@
...
+- Operations are now scoped to a particular client:
+    - `FileServiceClient`: This client handles account-level operations. This includes managing service properties and listing the shares within an account.
+    - `ShareClient`: The client handles operations for a particular share. This includes creating or deleting that share, as well as listing the directories within that share, and managing properties and metadata.
+    - `DirectoryClient`: The client handles operations for a particular directory. This includes creating or deleting that directory, as well as listing the files and subdirectories, and managing properties and metadata.
+    - `FileClient`: The client handles operations for a particular file. This includes creating or deleting that file, as well as upload and download data and managing properties.
+
+    These clients can be accessed by navigating down the client hierarchy, or instantiated directly using URLs to the resource (account, share, directory or file).
+    For full details on the new API, please see reference documentation.
```
Comment (chlowell) [inline sdk/storage/azure-storage-file/HISTORY.md]: Can this link to the reference documentation?

File: sdk/storage/azure-storage-file/HISTORY.md (hunk 10 above)
Comment (annatisch) [reply sdk/storage/azure-storage-file/HISTORY.md]: Oops!

File: sdk/storage/azure-storage-queue/HISTORY.md (hunk 9 above)
Comment (annatisch) [reply sdk/storage/azure-storage-queue/HISTORY.md]: Oops - copy+pasted a little too much ;)

**Pull request merged**